docs/pages/en/4.providers/imagekit.md

```diff
@@ -104,76 +84,73 @@ Add border to your images using modifier `border`. You can also set its width an
 ```html
 <nuxt-img
   provider="imagekit"
-  src="/medium_cafe_B1iTdD0C.jpg"
+  src="/default-image.jpg"
   width="300"
   :modifiers="{ border : '20_FF0000'}"
 />
 ```
 
+
 ### `rotate`
 
 Use `rotate` modifier to rotate your image. Possible values are-  `0`, `90`, `180`, `270`, `360`, and `auto`.
 
 ```html
 <nuxt-img
   provider="imagekit"
-  src="/medium_cafe_B1iTdD0C.jpg"
+  src="/default-image.jpg"
   :modifiers="{ rotate : 90 }"
 />
 ```
 
+
 ### `radius`
 
 Give rounded corners to your image using `radius`. Possible values are- positive integers and `max`.
 
 ```html
 <nuxt-img
   provider="imagekit"
-  src="/medium_cafe_B1iTdD0C.jpg"
+  src="/default-image.jpg"
   :modifiers="{ radius : 20}"
 />
 ```
 
+
 ### `bg`
 
 Specify background color and its opacity for your image using `bg` modifier. 
 
```

Comment:
In `bg`, if # is added to the hex code, does it still work?

Reply:
If the idea is to leave the current behavior implicit, then this is fine.

Reply:
Yeah, for `background` it is there because it would be a breaking change, for others `#` won't work